bandizip.sls

```diff
@@ -1,14 +1,27 @@
 #the provider only offers the download of the latest version
-{% set source_path = 'http://ca-dl.bandisoft.com/bandizip' %}
-{% set version = '6.21' %}
-
+{% set source_path_v6 = 'http://ca-dl.bandisoft.com/bandizip' %}
+{% set source_path_v7 = 'http://dl.bandisoft.com/bandizip.std' %}
+{% set version6 = '6.26' %}
+{% set version7 = '7.16' %}
+#
 bandizip:
-  '{{ version }}':
+  '{{ version6 }}':
```

Comment:
If the host only provides the latest version, then perhaps we treat this like chrome and use "latest" as the version.

Reply:
ok, done that.